app/config/settings_base.py

```diff
@@ -129,3 +130,8 @@
 # https://docs.djangoproject.com/en/5.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+# Cognito
+COGNITO_ENDPOINT_URL = env.str('COGNITO_ENDPOINT_URL', 'http://localhost:9229')
+COGNITO_POOL_ID = env.str('COGNITO_POOL_ID', 'local')
+COGNITO_FLAG_NAME = env.str('COGNITO_FLAG_NAME', 'dev:custom:managed_by_service')
```

Comment:
I think we should rename this to something `COGNITO_MANAGED_FLAG_NAME` as there might be other flags in the future!